community/rust/APKBUILD

```diff
@@ -58,6 +58,8 @@ subpackages="
 	cargo-doc:_cargo_doc:noarch
 	"
 source="https://static.rust-lang.org/dist/rustc-$pkgver-src.tar.gz
+	https://exp.exqa.de/alpine-rust/rust-armhf-$pkgver.tar.xz
+	https://exp.exqa.de/alpine-rust/rust-armv7-$pkgver.tar.xz
```

Comment:
This is on my private VPS for now, I'll have to get this mirrored on Alpine infra once this works.